pubsub/google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py

```diff
@@ -535,6 +543,15 @@ def _on_response(self, response):
             else:
                 self._messages_on_hold.put(message)
 
+        # Immediately (i.e. without waiting for the auto lease management)
```

Comment:
Can you please explain why this block was moved down? It seems like no control flow logic changes with this move.